users/views/settings/migration.py

```diff
@@ -34,17 +33,81 @@ def clean_alias(self):
             return self.alias_identity.actor_uri
 
     def form_valid(self, form):
-        self.identity.add_alias(form.cleaned_data["alias"])
-        messages.info(self.request, f"Alias to {form.alias_identity.handle} added")
+        if self.identity.has_moved():
+            messages.error(
+                self.request, "Alias update not allowed for a moved account."
+            )
+            return redirect(".")
```

Comment:
Should probably be `self.request.path` instead of "." (ditto for instances below)

Reply:
Ah, I didn't realize this was an existing convention. No need to change them, I'll get them all at once some day... maybe...